pkg/sfu/buffer/rtpstats.go

```diff
@@ -808,28 +850,27 @@ func (r *RTPStats) GetRtcpSenderReport(ssrc uint32, srDataExt *RTCPSenderReportD
 		return nil
 	}
 
-	// NTP timestamp in sender report from publisher side could have a different base,
-	// i. e. although it should be wall clock at time of send, have observed instances of older timer.
-	// It is not possible to accurately calculate current time in the NTP time base of the publisher side.
-	// So, using a smoothed version of one way delay for use in sender reports.
-	now := time.Now()
+	// construct current time based on monotonic clock
+	timeSinceFirst := time.Since(r.firstTime)
+	now := r.firstTime.Add(timeSinceFirst)
```

Comment:
Done this way to ensure that monotonic clock is used compared to the first packet, i. e. first packet time acts as the base and then everything else is an offset.